apps/portal/src/App.js

```diff
@@ -854,7 +853,7 @@ export default class App extends React.Component {
             return {
                 page: 'supportError'
             };
-        } else if (path === 'recommendations' && hasRecommendations({site})) {
```

Comment:
Ah, this the culprit for no the modal not triggering detailed in the referenced issue?

Reply:
I thought so, but on further investigation, I think he was reporting a separate bug in ghost_head that caused no loading of portal. (Separate PR filed.)